TranslatorService.py

```diff
@@ -0,0 +1,27 @@
+import requests
+
+def translate_text(text, source_lang, target_lang):
+    api_url = "https://libretranslate.de/translate"
+    
+    params = {
+        'q': text,
+        'source': source_lang,
+        'target': target_lang
+    }
+
+    response = requests.post(api_url, data=params)
+
+    if response.status_code == 200:
+        translation = response.json()['translatedText']
+        return translation
+    else:
+        return f"Error {response.status_code}: {response.text}"

```

Comment:
all functional, but it doesn't reflect the architecture discussed last week (feel free to check the slides/recording again). You are hard coupling your API logic with the service logic here, you want to create an abstract service, a mocked service and api service implementing the abstract service's methods.